### ISSUE TITLE
Dashboard UI Revisions

### ISSUE THREAD

**Issue by tommerty** [usesend/useSend#80]
- Sidebar
  - [ ] Improve the colors and text
  - [ ] Maybe look into Shads new sidebars, see how I can make it more dynamic
- Page heading 
  - [ ] Shifts depending on if the toggle for 7 days/30 days for example is present. See if I can get a more globalized uniform solution
- [ ] General themeing improvemts and uniforming

@KMKoushik I can't seem to assign myself to this, if you could assign me and I'll start up my patch in reference

**Comment by KMKoushik** [comment]
@tommerty assigned to you! thanks!

**Comment by tommerty** [comment]
Initial start to the UI overhaul, starting with the sidebar. Uses [ShadCNs sidebar component](https://ui.shadcn.com/docs/components/sidebar) for the most part.

https://github.com/user-attachments/assets/8a6e3e15-947a-4f66-b704-b9fc2afb8e97

**Comment by KMKoushik** [comment]
looking great

**Comment by tommerty** [comment]
Leaving this on hold until aws isn't required for a localdev environment as since an OS change I've been having problems getting an env set up.

TLDR waiting for https://github.com/KMKoushik/local-ses-sns

**Comment by KMKoushik** [comment]
@tommerty , implemented this in latest rebrand. thanks for the inputs. i used shadcn as well
